### DISCUSSION TITLE
Add support for additional file formats

### DISCUSSION THREAD

**Discussion by LeoKlaus** [LeoKlaus/Paperparrot#66] [Feature Requests]
Like docx, xlsx, rtf,..

This should be hidden behind a toggle when connected to a server, as it requires the Tika extension serverside.

**Comment by JigSawFr** [comment]
@LeoKlaus not possible to know if tika / gotenberg are enabled via API?

**Reply by LeoKlaus** [reply]
Not to my knowledge. The documentation doesn't mention anything. I'll have to setup a test instance and check.

**Comment by LeoKlaus** [comment]
Implemented this. Will be included with the next update.
